book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
```

Comment:
- [ ] I would highly recommend refining this method for adding rentals by having it accept both date and person parameters. 📆👤 Then, when you call the Rental.new constructor, you can pass self as the book parameter. This approach will effectively establish a "has-many" association between your books and rentals. 📚👥
- [ ] Here's why it's a great idea: When you modify your method to accept the date and person parameters, you're enhancing the flexibility and functionality of your code. 🛠️ It enables you to create rentals with specific information and links them directly to the book they belong to. By passing self to the Rental.new constructor, you establish a clear relationship between books and their respective rentals. 🧩 This organized structure simplifies operations and makes it easier to manage and query rentals associated with a particular book. 📊📖 Plus, it aligns with the "has-many" relationship concept, which is a solid foundation for effective data management in your application. 🤝📈

```suggestion
    def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end
```